### PR TITLE
Use new product builder for Leap 15

### PR DESCRIPTION
**Pull request by Vogtinator**
This is basically a combination of #17 and #18, so that the product builder code can be tested for Leap 15 to get some images out.

### CODE REVIEW

File: doitall.sh [hunk 1]
```diff
@@ -8,9 +8,9 @@ test -n "$proj" || proj=Factory
 test -n "$repo" || repo=standard
 
 if [ "$proj" = "Factory" ]; then
-    product=000product
```
Comment (DimStar77) [inline doitall.sh]: That feels like we break on Factory:PowerPC, Factory:ARM, Factory:zsystems

Comment (Vogtinator) [reply doitall.sh]: Those use the product builder already?

Comment (jberry-suse) [reply doitall.sh]: There is openSUSE/osc-plugin-factory#1112 for zsystems, but not sure what is staged vs working.

Comment (Vogtinator) [reply doitall.sh]: @DimStar77 That diff only happens if diffing your pr vs. mine, so it would be already broken in master if that was the case?

Comment (Vogtinator) [reply doitall.sh]: Ah, so it was meant to be the other way around, only use 000product for Factory itself but not the ports?

Comment (DimStar77) [reply doitall.sh]: as for the ttm submission: zSystems is 'in limbo' - the old prod builder doesn't produce anything usable, the new one needs more work on ttm (openSUSE/osc-plugin-factory#1112  is a beginning, but we know it won't handle the _multibuild setup created by the new prod builder service)

Comment (Vogtinator) [reply doitall.sh]: I replaced it with the usual switch-case statement from the other files now, so that should be fine.

Comment (DimStar77) [reply doitall.sh]: > Ah, so it was meant to be the other way around, only use 000product for Factory itself but not the ports?

My initial pr to get started? yes, that was breaking 'my things first' - then fix and follow through to the ports.

But happy you're testing this in Leap now :)

**Pull request merged**